sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -334,6 +336,8 @@ PARAM_TRAITS_SPEC(device, reference_count, cl_uint)
 PARAM_TRAITS_SPEC(device, max_num_sub_groups, cl_uint)
 PARAM_TRAITS_SPEC(device, sub_group_independent_forward_progress, bool)
 PARAM_TRAITS_SPEC(device, sub_group_sizes, vector_class<size_t>)
+PARAM_TRAITS_SPEC(device, kernel_kernel_pipe_support, bool)
+PARAM_TRAITS_SPEC(device, spirv_version, string_class)
```

Comment:
I cannot find the implementation of this trait. How it actually maps to OpenCL device info calls?

Reply:
It's just piGetDeviceInfo(..., param, ...), which is mapped to clGetDeviceInfo(..., param, ...), where "param" is CL_DEVICE_IL_VERSION. See device_info.hpp, lines 70-85.